src/app/features/management/user-management-view/edit-user-view/edit-user-view.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="view" [appShowLoadingOverlay]="updatingUser.loadingChange() | async">
+  <div class="heading">
+    <h1 i18n>User: {{form.controls.firstName}} {{form.controls.lastName}}</h1>
+    <button mat-button type="button" (click)="changePassword()" i18n>
+      Change Password
+    </button>
```

Comment:
see comments from last review. please move to bottom buttons (middle position) as this button is easy to miss. i even missed it as well.